src/DynamicRupture/Output/Builders/GeoGeometryBuilder.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef SEISSOL_GEO_GEOMETRY_BUILDER_HPP
```

Comment:
Is the first `GEO` on purpose?

Reply:
Renamed to `SEISSOL_DR_INTEGRATED_OUTPUT_BUILDER_HPP`. 

In general, I renamed the entire class name. Right now it is called `IntegratedOutputBuilder`